packages/headless/src/features/configuration/configuration-slice.test.ts

```diff
@@ -235,6 +238,7 @@ describe('configuration slice', () => {
           apiBaseUrl: 'http://test.com/search',
           locale: 'fr-CA',
           timezone: 'Africa/Johannesburg',
+          authenticationProviders: ['theNewProvider'],
```

Comment:
What's the difference between theProvider and theNewProvider

Reply:
That specific test is about verifying the work the reducer does when changing from an existing state to another.

So going from `theProvider` to `theNewProvider`.